src/evergreen/base.py

```diff
@@ -2,47 +2,61 @@
 """Task representation of evergreen."""
 from __future__ import absolute_import
 
-from evergreen.util import parse_evergreen_datetime, parse_evergreen_date
+from evergreen.util import parse_evergreen_datetime, parse_evergreen_date, \
+    parse_evergreen_short_datetime
 
 
-def evg_attrib(attrib_name, type_fn=None):
+def evg_attrib(attrib_name, type_fn=None, default=None):
```

Comment:
I'm not sure how I feel about injecting a default value when the actual API didn't have a value. I think I would rather have the user of this code deal with missing values. It seems like there could be cases where the user might want to behave differently if the value is missing.